revolv/templates/base/partials/dashboard_project.html

```diff
@@ -52,11 +52,34 @@ <h2 class="small-10 small-offset-1 columns">{{project.tagline}}</h2>
     </div>
     <div class="content funding{% if not project.is_completed %} active{% endif %}" id="funding-{{project.pk}}">
         <div class="row donation-stats">
-            <div class="days-remaining small-6 medium-3 large-2 columns">
-                <div class="num-days-remaining">{{project.days_left}}</div>
-                <div class="days-remaining-label">day{{project.days_left|pluralize}} left</div>
+            <div class="small-12 medium-12 large-2 large-offset-1 columns" style="height:140px">
+                <div class="funding-progress-container">
+                    <div class="statistics-graphics-container">
+                        <svg class="svg-graphics-container"></svg>
+                        <svg class="internal-graphics-container"></svg>
+                        <div class="text-container">
+                            <p class="percentage-text">{{ project.percent_complete }}%</p>
+                            <p class="funded">FUNDED</p>
+                        </div>
+                        <div class="percentage-container hide">{{ project.partial_completeness_as_js }}</div>
+                    </div>
+                </div>
             </div>
-            <div class="donor-stats-table small-6 medium-3 large-4 columns end">
+            <div class="donor-stats-table small-12 medium-6 large-3 columns">
+                <div class="row donor-stats-table-row">
```

Comment:
should call this `funding-stats-table-row` then add a css rule rather than calling both tables the same thing, I think. Same for `donor-stats` above